src/main/java/org/checkerframework/checker/https/HTTPSAnnotatedTypeFactory.java

```diff
@@ -0,0 +1,55 @@
+package org.checkerframework.checker.https;
+import com.sun.source.tree.*;
+import org.checkerframework.common.basetype.BaseAnnotatedTypeFactory;
+import org.checkerframework.common.basetype.BaseTypeChecker;
+import org.checkerframework.framework.qual.TypeUseLocation;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+import org.checkerframework.framework.util.defaults.QualifierDefaults;
+import org.checkerframework.javacutil.AnnotationBuilder;
+import org.checkerframework.checker.https.qual.HTTPS;
+
+import javax.lang.model.element.AnnotationMirror;
+import java.lang.annotation.Annotation;
+import java.util.Set;
+
+/** @checker_framework.manual #HTTPSChecker HTTPS Checker*/
+public class HTTPSAnnotatedTypeFactory extends BaseAnnotatedTypeFactory {
+
+    /** The @HTTPS annotation. */
+    private final AnnotationMirror HTTPS;
+
+    /** Create a HTTPSAnnotatedTypeFactory. */
```

Comment:
This Javadoc doesn't tell the reader anything. Of course the constructor creates a new instance of the class - that's what constructors always do. If there's nothing interesting to say about a method, it's better to say nothing. Use comments to express interesting choices about the design, invariants about arguments, or anything else that isn't obvious to a reader right away.